svo (alias and defence functions).xml

```diff
@@ -1489,7 +1489,8 @@ end
     on = "You deftly shape the wall of light into a barrier surrounding yourself.",
     onr = [[^You strum .+, and a prismatic barrier forms around you\.$]],
     def = "You are standing within a prismatic barrier.",
-    off = {"Your prismatic barrier dissolves into nothing.", "The stream hits your prismatic barrier, shattering it.", "The breath weapon rips apart your prismatic barrier.", "The breath weapon rips through both your shield and prismatic barrier.", "The spear shatters your prismatic barrier."}})
+    off = {"Your prismatic barrier dissolves into nothing.", "The stream hits your prismatic barrier, shattering it.", "The breath weapon rips apart your prismatic barrier.", "The breath weapon rips through both your shield and prismatic barrier.", "The spear shatters your prismatic barrier."},
+    offr = "Raising (?:his|her) arm dramatically, .+ brings forth a single great chime from (?:her|his) crier's bell. The sound rings out, it's powerful vibration collapsing the prismatic barrier surrounding you."})
```

Comment:
Haven't tried but it looks like you have an extra `}` in there

Reply:
I think it closes the "{ type = " bracket earlier in the code, at least from what i can read